### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by sanchit-sinha**
invalid_email
valid_email

### CODE REVIEW

File: src/views/examples/Login.js [hunk 1]
```diff
@@ -31,60 +31,55 @@ import {
...
+  const [validemail, setValidemail] = useState(false);
+
+  const email_verification = async (target_email) => {
+    var class_valid = "is-valid form-control";
+    var class_invalid = "is-invalid form-control";
+
+    target_email.className = class_invalid;
+    if (
+      /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@iitk.ac.in/.test(target_email.value)
+    ) {
```
Comment (mridul911) [inline src/views/examples/Login.js]: Great!

**Review by mridul911 — APPROVED** [review]
Great work! Can you just remove some unnecessary things from login page and squash all commits into one.

**Comment by sanchit-sinha** [comment]
Updated Login UI as per https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#pullrequestreview-677819144
updatedui

**Comment by kartikcode** [comment]
Remove the option of Create New Account. Also, maybe we can have SPO logo and title instead of the Argon one.

**Comment by sanchit-sinha** [comment]
Hey, we are using the `username` and the `password` of the user as the POST request's credential(in the backend). So, shall I update the Login UI with the following changes:

- Using the `IITK Username` field instead of the `IITK Email ID`, for logging in  
- Removing the  Email validator
- Updating the dummy API

Suggest some other changes if required!

**Comment by sanchit-sinha** [comment]
Made required changes as per https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#issuecomment-857786291 and https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#issuecomment-859839392
Screenshot from 2021-06-13 11-24-54

**Comment by sanchit-sinha** [comment]
I have removed unnecessary information from the footbar, navbar and added a temporary logo. Do I need to revert these changes as per #10 or is it fine ??

**Pull request merged**